doc/source/user_guide/numpy_images.rst

```diff
@@ -168,43 +167,43 @@ These conventions are summarized below:
   =========================   ========================================
 
 
-Many functions in scikit-image operate on 3D images directly:
+Many functions in ``scikit-image`` can operate on 3D images directly:
 
     >>> im3d = np.random.rand(100, 1000, 1000)
     >>> from skimage import morphology
     >>> from scipy import ndimage as ndi
     >>> seeds = ndi.label(im3d < 0.1)[0]
     >>> ws = morphology.watershed(im3d, seeds)
 
-In many cases,
-the third imaging dimension has lower resolution than the other two.
-Some scikit-image functions provide a ``spacing`` keyword argument
-to process these images:
+In many cases, however, the third spatial dimension has lower resolution
+than the other two. Some ``scikit-image`` functions provide a ``spacing``
+keyword argument to help handling this situation:
```

Comment:
to help handle

Reply:
situation -> kind of data? "Situation" makes it sound more unfortunate than it really is. =P